src/openforms/templates/includes/forms/field_wrapper.html

```diff
@@ -0,0 +1,15 @@
+{# Takes a 'field' context argument, which should be an instance of a BoundField #}
+<div class="openforms-form-control">
+    <div class="openforms-form-control openforms-form-control--{{ type|default:'textfield' }}">
+        <label for="{{ field.auto_id }}" class="openforms-label">
+            {{ field.label }}
+        </label>
+        <div>
+            {{ field }}
+        </div>
+        {% if field.help_text %}<div class="openforms-help-text">{{ field.help_text }}</div>{% endif %}
```

Comment:
I wonder if we should use something like `aria-describedby` so that a screen reader knows to which input the help-text corresponds?

Reply:
I looked into it but it's quite messy since we don't have pure template based form rendering yet. This would require us to set the `aria-describedby` attribute in python on the form field widget, but it also needs to know if the label/help text are going to be rendered or not since multiple values apply (and I think both label and help text describe the input).